README.md

```diff
@@ -25,29 +25,29 @@ public class ElizaServiceClient(
         request,
         headers,
         MethodSpec(
-            "buf.connect.demo.eliza.v1.ElizaService/Say",
-            buf.connect.demo.eliza.v1.SayRequest::class,
-            buf.connect.demo.eliza.v1.SayResponse::class
+            "connectrpc.eliza.eliza.v1.ElizaService/Say",
```

Comment:
This doesn't look right - I believe this is meant to reflect the generated code from ./examples/generated-google-java/src/main/kotlin/com/connectrpc/eliza/v1/ElizaServiceClient.kt. In this case, it should be:

```kotlin
MethodSpec(
  "connectrpc.eliza.v1.ElizaService/Say",
  com.connectrpc.eliza.v1.SayRequest::class,
  com.connectrpc.eliza.v1.SayResponse::class,
),
```

instead of:
```kotlin
MethodSpec(
  "connectrpc.eliza.eliza.v1.ElizaService/Say",
  connectrpc.eliza.eliza.v1.SayRequest::class,
  connectrpc.eliza.eliza.v1.SayResponse::class
)
```